pvlib/bifacial/utils.py

```diff
@@ -169,3 +170,226 @@ def _vf_ground_sky_2d(x, rotation, gcr, pitch, height, max_rows=10):
     np.clip(next_edge, a_min=0., a_max=None, out=next_edge)
     vf = np.sum(next_edge, axis=-1) / 2
     return vf
+
+
+def vf_ground_sky_2d_integ(surface_tilt, surface_azimuth, gcr, height,
+                           pitch, max_rows=10, npoints=100, vectorize=False):
```

Comment:
Can we nix `surface_azimuth` here?  I think it was originally used when the function also took `axis_azimuth` and a tracker rotation was calculated, but when we removed that calculation (in favor of the "abuse" mentioned in a comment below) it looks like we didn't think to remove `surface_azimuth` along with it.